### ISSUE TITLE
[Translation tool] id_ID

### ISSUE THREAD

**Issue by diankur313** [briannesbitt/Carbon#1929]
|English|id_ID|comments|
|--|--|--|
|2 hours from now|dalam 2 jam|dua jam dari sekarang because its seems starting point to other point from 2 hours range. the "dalam dua jam" maybe correct, but its ambiguos. "now" its mean "sekarang". its clear if that mean "dua jam dari sekarang"|

**Comment by kylekatarnls** [comment]
Hi @diankur313 can you tell me how you would translate "in 2 hours" in the sentence:
> We'll lunch in 2 hours

It would be the kind of meaning the "from now" translation aims to be for.

**Comment by a-wip0** [comment]
> We'll lunch in 2 hours

with "from now" aim
> Kita akan makan siang 2 jam dari sekarang

without "from now" aim
> Kita akan makan siang dalam 2 jam

**Comment by a-wip0** [comment]
@diankur313 is right it could be ambiguos, i think of 2 situation as example:

1. Now time is 10.00, and you tell your friends "We'll lunch in 2 hours", you and your friends will start eat at 12.00, in this case the meaning of "We'll lunch in 2 hours" is "Kita akan makan siang dua jam dari sekarang".
2. Now time is 10.00 and you tell your friends  "We'll lunch in 2 hours", you and your friends are now having lunch and will spend next 2 hours to eat finish at 12.00, in this case the meaning of "We'll lunch in 2 hours" is "kita akan makan siang dalam 2 jam." or "kita akan makan siang selama 2 jam."
"dalam" and "selama" in this example have same meaning.

**Comment by kylekatarnls** [comment]
OK. I'm very surprised that http://cldr.unicode.org/ is wrong on that, ID has more than 200 millions of speakers and CLDR is the most widely used reference.

But the aim is clearly "start in 2 hours", not about a duration. So I will change it.

**Comment by kylekatarnls** [comment]
Hi @a-wip0 @diankur313 @nafiesl can you confirm "x dari sekarang" is better than "dalam x" in all the following examples (some common Carbon usages):

```
Meeting "Title": 15 menit dari sekarang
```
```
Birday of Bob: 26 hari dari sekarang
```
```
Session expiration: 3 menit dan 42 detik dari sekarang
```

If so, I will merge #1936

**Comment by nafiesl** [comment]
From me, using "3 jam dari sekarang" makes more sense then using "dalam 3 jam". So, yes, it's better.

**Comment by kylekatarnls** [comment]
Thanks!

**Comment by kylekatarnls** [comment]
Thanks! It will be released in 2.27
